seed_data/challenges/computer-science.json

```diff
@@ -137,8 +158,13 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Waypoint: Aprende sobre Seguridad Informática",
+      "descriptionEs": [
+        "¡Ya casi completamos el curso de Introducción a la Informática!",
+        "Aprenderemos sobre una de más importante invención del siglo 20 - las hojas de cálculo.",
+        "También aprenderemos sobre la Seguridad Informática y sobre algunas de las vulnerabilidades más comúnes en los sistemas de software."
```

Comment:
Just little detail on this, I would remove the second "sobre":
"También aprenderemos sobre la Seguridad Informática y algunas de las vulnerabilidades más comúnes en los sistemas de software."